app/components/app_consent_component.rb

```diff
@@ -43,11 +44,10 @@ def contact_parent_or_guardian_link(consents)
     link_to(
       "Contact #{consent.parent_name} (the #{role} who refused)",
       session_patient_manage_consent_path(
-        session,
-        patient,
-        @route,
-        consent.id,
-        :agree
+        session_id: session,
```

Comment:
Hmm, does this work as-is? Maybe for consistency:

```suggestion
        session_id: session.id,
```

Or if the var was an id all along:

```suggestion
        session_id:,
```

Reply:
In my experimentation, `session_id` can be set to either `session` or `session.id`, so the original code above should work, ya. However, like you suggest, I've been trying to standardise on `session.id`, I probably just missed that one. It's a shame that `session: session` doesn't work as that could've been nicely abbreviated as `session:`